source/analytics/reports/jurisdictions/montana/v16 CDEpiWeekly/README.md

```diff
@@ -0,0 +1,19 @@
+# v16 CdEpiWeekly Morbidity and Mortality Weekly Report
+
+Recreating the v16 CdEpiWeekly Morbidity and Mortality Weekly Report (MMRW) for Montana on Metabase using the Reporting Database. This report provides the aggregate count of most reportable conditions for a chosen week in comparison to the same week in the previous three years.
+
+## v16: Report Details
+
+Item: CdEpiWeekly
+
+App Type: Task Server 
+
+Run time: 20 seconds
+
+Run interval: 7pm Wed/Thurs
+
+Database: MIDIS RDB
+
+Tables: BmirdCases, CaseCounts, Conditions, ConfirmationMethods, ConfirmationMethodGroups, DPatients, FVarPams, GenericCases, HepatitisCases, Investigations, LdfData, MeaslesCases, NotificationEvents, PertussisCases, RdbDates, RubellaCases
```

Comment:
Please add a detail if these tables are from the RDB. As some reports have dependency on other DBs such as ODSE, SRTE and any other jurisdiction specific data sources.